tapchannel/aux_invoice_manager.go

```diff
@@ -42,6 +42,12 @@ type RfqManager interface {
 	// node and have been requested by our peers. These quotes are
 	// exclusively available to our node for the sale of assets.
 	LocalAcceptedSellQuotes() rfq.SellAcceptMap
+
+	// AssetMatchesSpecifier checks if the provided asset satisfies the
+	// provided specifier. If the specifier includes a group key, we will
+	// check if the asset belongs to that group.
+	AssetMatchesSpecifier(ctx context.Context, specifier asset.Specifier,
```

Comment:
Seems the `rfq` package is already imported here, so we can embed that interface. Mainly a question re style.